### ISSUE TITLE
Seafile - system not support yet

### ISSUE THREAD

**Issue by chaosbunker** [j8r/DPlatform#44]
I tried on both Ubuntu 16.04 and debian 9 .. unfortunately for both i got the message the system is not supported yet. What system is supported?

I am surprised Ubuntu and debian are not supported - i have run it on both in the past. 

What needs to happen for seafile to be supported?:-)

**Comment by j8r** [comment]
You have probably tried to install Seafile with MariaDB. The point is that the two entities of Seafile, the chinese Seafile Ltd. and the german Seafile GmbH, have separated themselves to follow their own ways.

We have used the installer of the germans to install Seafile, but it's Syncwerk now. The chinese have forked them to [this repo](https://github.com/haiwen/seafile-server-installer).
I've updated the links to apply this changes, espcially new distributions have been release since.
